src/site/content/en/fast/signed-exchanges-webpackager/index.md

```diff
@@ -230,238 +265,100 @@ used with signed exchanges.
-    `CanSignHttpExchanges` certificate, you must generate the CBOR-encoded
-    certificate yourself.  
-
-
-### Setup up Web Packager for production use
-
-
-
-1. Follow steps 1 and 2 of [Setup the Web Packager Server for
-   testing](/signed-exchanges-webpackager/#setup-the-web-packager-server-for-testing).
+   [PEM](https://en.wikipedia.org/wiki/X.509#Certificate_filename_extensions) is
+   a file format that is commonly used as a "container" for storing multiple
+   certificates.
 
 2. Open `webpkgserver.toml` with the editor of your choice and make the
    following changes:
-    *   Change the line `PEMFile = cert.pem` to `PEMFile = cert.cbor`.
-
-
-### Upload the `CanSignHttpExchanges` certificate
-
-
-
-1.  Follow steps 1 through 4 of [Upload the self-signed
-    certificate](/signed-exchanges-webpackager/#upload-the-self-signed-certificate) to upload `cert.cbor`. 
-
-3. Adjust your server config to serve `cert.cbor` using the `Content-Type:
-   application/cert-chain+cbor` response header.
-
-    If this header is not set, you will see the following error when you inspect the
-    SXG in DevTools: `Content type of cert-url must be application/cert-chain+cbor.
-    Actual content type: text/html`.
-
-4. Restart `webpkgserver`.
-
-    ```shell
-    webpkgserver
-    ```
+    *   Change the line `PEMFile = cert.pem` to reflect the location of the PEM
+        file containing your full certificate chain.
+    *   Change the line `KeyFile = 'priv.key'` to reflect the location of the
+        private key corresponding to your PEM File.
+    *   Change the line `Domain = 'example.org'` to reflect your site.
+    *   (Optional) To have `webpkgserver` auto-renew the SXG certificate every
+        90 days, configure the options in the `[SXG.ACME]` section of
+        `webpkgserver.toml`. This option only applies to sites with a DigiCert
+        ACME account setup.
+
+3.  Configure your edge server to forward traffic to the `webpkgserver`
+    instance.
+
+    There are two primary types of requests handled by `webpkgserver`: requests
+    for SXGs (which are served by the `/priv/doc/` endpoint) and requests for
+    the SXG certificate (which are served by the `/webpkg/cert/` endpoint). The
+    URL rewriting rules for each of these request types varies slightly. For
+    more information, see [Running behind front end edge
+    server](https://github.com/google/webpackager/blob/master/cmd/webpkgserver/README.md#running-behind-front-end-edge-server).
+
+    **Note:**
+
+    By default, `webpkgserver` serves the SXG certificate at
+    `/webpkg/cert/$CERT_HASH`—for example,
+    `/webpkg/cert/-0QmE0gvoedn92gtwI3s7On9zPevJGm5pn2RYhpZxgY`.
+    To determine the hash corresponding to your SXG certificate, look in the cache
+    directory (that is, `/tmp/webpkg`). The filenames of the certificates in
+    this directory follow the convention `$CERT_HASH.pem` and `$CERT_HASH.ocsp`.
```

Comment:
@twifkak @banaag is there a better way to find out this hash? I feel like I should be able to do something like `openssl x509 -noout -pubkey -in cert.pem | openssl pkey -pubin -outform der | openssl dgst -sha256 -binary | base64` (but that's a different hash).

Reply:
@khempenius I think looking in `/tmp/webpkg` is the easiest way.  If you want to know how this digest is computed in code, you can find it here: https://github.com/google/webpackager/blob/master/certchain/raw_chain.go#L114.

Reply:
This works:

```
$ openssl base64 -in /tmp/webpkg/ZxrUJSsT_TazV461y728u3BZSWEjJ3l9kbub6jVI-9k.pem -d | openssl dgst -sha256 -binary | base64 | tr '/+' '_-' | tr -d =
ZxrUJSsT_TazV461y728u3BZSWEjJ3l9kbub6jVI-9k
```